fvm/bootstrap.go

```diff
@@ -678,8 +678,8 @@ transaction(clusterWeights: [{String: UInt64}]) {
 		randomSource: %s,
 		collectorClusters: clusters,
         // NOTE: clusterQCs and dkgPubKeys are empty because these initial values are not used
-		clusterQCs: [],
-		dkgPubKeys: [],
+		clusterQCs: [] as [FlowClusterQC.ClusterQC],
+		dkgKeys: [] as [String],
```

Comment:
Was this change necessary? It shouldn't be

Reply:
Might not be for cadence 0.23. Let me check

Reply:
removed [6d883fd](https://github.com/onflow/flow-go/pull/1605/commits/6d883fd26a95beabcc6c198247ce1e7d1edef6c4)